### DISCUSSION TITLE
Window palette issue in eclipse

### DISCUSSION THREAD

**Discussion by PrakashIrom** [eclipse/windowbuilder#333] [Q&A]
![Screenshot (5)](https://user-images.githubusercontent.com/114992036/195610532-ee8a3d77-91c3-40c6-b19b-ce298b87e16f.png)I 
I downloaded window builder in my eclipse and when I click on the design tab my palette is not showing properly, it has no max,min and close button and I cant use any of the components,layouts,containers,etc... Can you guys help me out with this issue please

**Comment by wimjongman** [comment]
You have dragged the components and properties. See video. Use double-click on the title to minimize/maximize

https://user-images.githubusercontent.com/180969/195672074-d55d9539-0d24-48f5-be53-7d1db5344c2f.mp4

**Reply by PrakashIrom** [reply]
https://user-images.githubusercontent.com/114992036/195752265-1faec699-8e77-42a6-bfbe-002d42bf582f.mp4

Sir you can see it from the video that when I clicked on the Jlabel, I can place it on my window panel but I can't edit or put my label.It says new label only and when I right clicked on it the cursor didn't even come out. I think I missed something to download to be able to use the window builder's features.

**Reply by wimjongman** [reply]
You can change the text of the label in the source, by pressing SPACE when the item is focussed or by using the properties.

**Reply by wimjongman** [reply]
https://help.eclipse.org/latest/topic/org.eclipse.wb.doc.user/html/index.html?cp=89

**Comment by PrakashIrom** [comment]
![copy ss5](https://user-images.githubusercontent.com/114992036/195752999-2f559e44-7642-4715-aefe-2f7f2ce3d903.png)
And as you can see it from the photo(the circled part),the close, max, min option is missed.

**Reply by wimjongman** [reply]
Go ahead and post your source.

**Reply by PrakashIrom** [reply]
I am a new to programming I don't know how to upload sir

**Reply by PrakashIrom** [reply]
@wimjongman how can I contact you sir? I need to complete a college project using the window builder and which due date is on 20th oct. I even watched several youtube videos but I didn't find any videos related with my issue. Can you please help me out??

**Reply by PrakashIrom** [reply]
@wimjongman sir I uploaded the source and my repository name is Window-Builder-Eclipse please help me out

**Reply by wimjongman** [reply]
Sorry, you are incomplete. Please attach the source to the conversation.

Please note that I am not responsible for your deadline. 

Also, Windowbuilder is a tool. You can type in the code directly.

I am working with Java 17. What Java version are you using?

**Reply by PrakashIrom** [reply]
[src.zip](https://github.com/eclipse/windowbuilder/files/9800746/src.zip)
Sir am I attaching the right file please do check?? I am using Java 8.

**Reply by wimjongman** [reply]
Java 8 is not supported. I doubt that you use it.

Please open the error log view (CTRL+3, then search 'error log') export the log and attach it here.

**Reply by wimjongman** [reply]
> [src.zip](https://github.com/eclipse/windowbuilder/files/9800746/src.zip) Sir am I attaching the right file please do check?? I am using Java 8.

No this is not the source. Your source is exactly one file.

**Reply by PrakashIrom** [reply]
[error log.log](https://github.com/eclipse/windowbuilder/files/9800938/error.log.log)
Sir I am indeed using java 8 but I don't know how if it's not supported.

**Reply by merks** [reply]
The src.zip is the source of the JDK not your source file...

The error log says ```java.version=17.0.4.1``` so the IDE is running Java 17.

I wonder what this is telling me from the log:
```
org.eclipse.wb.internal.core.utils.exception.DesignerException: 103 (Unknown GUI toolkit). kasu.Window
	at org.eclipse.wb.internal.core.parser.JavaInfoParser.prepareParseContext(JavaInfoParser.java:1204)
```

**Reply by PrakashIrom** [reply]
@merks sir I think this is it?? Please check
[srce.zip](https://github.com/eclipse/windowbuilder/files/9801145/srce.zip)

**Reply by PrakashIrom** [reply]
I don't know about the version I downloaded the Java SE development kit 8

**Reply by wimjongman** [reply]
This source looks good. I don't have any issues with it.

Please install a fresh copy of Eclipse without any other plugins beside WindowBuilder. I see a lot of other plugin action in your log:

<details>
!ENTRY org.eclipse.equinox.p2.repository 2 0 2022-10-17 11:27:37.872
!MESSAGE Using unsafe http transport to retrieve http://www.genuitec.com/updates/codemix/ci/, see CVE-2021-41033. Consider using https instead.

!ENTRY org.eclipse.equinox.p2.repository 2 0 2022-10-17 11:27:40.420
!MESSAGE Using unsafe http transport to retrieve http://downloads.genuitec.com/downloads/products/codemix/updates/cm20200722, see CVE-2021-41033. Consider using https instead.

!ENTRY org.eclipse.equinox.p2.repository 2 0 2022-10-17 11:27:54.933
!MESSAGE Using unsafe http transport to retrieve http://www.genuitec.com/updates/codemix/ci/, see CVE-2021-41033. Consider using https instead.

!ENTRY org.eclipse.equinox.p2.repository 2 0 2022-10-17 11:27:57.812
!MESSAGE Using unsafe http transport to retrieve http://www.genuitec.com/updates/devstyle/ci/, see CVE-2021-41033. Consider using https instead.

!ENTRY org.eclipse.equinox.p2.repository 2 0 2022-10-17 11:27:59.777
!MESSAGE Using unsafe http transport to retrieve http://downloads.genuitec.com/downloads/products/devstyle/updates/ds20220613a, see CVE-2021-41033. Consider using https instead.

!ENTRY org.eclipse.equinox.p2.repository 2 0 2022-10-17 11:28:05.501
!MESSAGE Using unsafe http transport to retrieve http://downloads.genuitec.com/downloads/products/codetogether/updates/ct20220914/content.xml.xz, see CVE-2021-41033. Consider using https instead.
</details>

**Reply by PrakashIrom** [reply]
I just downloaded a new eclipse(2022/09) and window builder but when I created a new Jframe using Java-SE17 and clicked on design tab, it says unknown gui toolkit error and then I changed it to Java-SE1.7, the error is gone but the same issue I faced it again. This issue is more complicated than I expected.
